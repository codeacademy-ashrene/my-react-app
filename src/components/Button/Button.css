.Button {
  float: left;
  color: "red";
}