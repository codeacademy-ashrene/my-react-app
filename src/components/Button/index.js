import React, { Component } from 'react';
import './Button.css';

class Button extends Component {
  render() {
    return (
      <div>
        <button type="button" className="Button">Click me!</button>
      </div>
    );
  }
}

export default Button;
