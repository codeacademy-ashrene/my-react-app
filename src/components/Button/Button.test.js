import React from 'react';
import Button from '.';
import renderer from 'react-test-renderer';

describe('Button', () => {
  it('should render a button', () => {
    const tree = renderer.create(<Button/>).toJSON();
    expect(tree).toMatchSnapshot();

  });
});