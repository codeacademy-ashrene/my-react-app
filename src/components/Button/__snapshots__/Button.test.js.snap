// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render a button 1`] = `
<div>
  <button
    className="Button"
    type="button"
  >
    Click me!
  </button>
</div>
`;
